itunes/commands.go

```diff
@@ -201,13 +213,64 @@ func listMusics(selectType SelectType) (string, error) {
 	return list, nil
 }
 
+func status(c *cli.Context) error {
+	state, err := mack.Tell("iTunes", "get player state as string")
+
+	if err != nil {
+		return fmt.Errorf("cannot get iTunes status: %s", err)
+	}
+	fmt.Println("iTunes is currently", state)
+
+	if state == "playing" {
+		artist, _ := mack.Tell("iTunes", "get artist of current track as string")
+		track, _ := mack.Tell("iTunes", "get name of current track as string")
+
+		fmt.Println("Current Track ", artist, ": ", track)
+	}
+
+	return nil
+}
+
+func shuffle(c *cli.Context) error {
+	if c.NArg() > 1 {
+		cli.ShowCommandHelp(c, "shuffle")
+		return fmt.Errorf("\ninvalid arguments number")
+	}
+
+	var err error
+	if c.NArg() == 1 {
```

Comment:
In the case of `c.NArg() == 1`, it is good to validate whether the first argument is one of `ON`, `on`, `OFF` or `off`.
And if an invalid argument is passed, `shuffle` should return an error.